lib/generators/scim_rails/templates/initializer.rb

```diff
@@ -157,4 +157,110 @@
   # Set group_destroy_method to a method on the Group model
   # to be called on a destroy request
   # config.group_destroy_method = :destroy!
+
+  # /Schemas settings.
+  # This configure is not used in /Users and /Groups for now.
+  # Configure this only when you need Schemas endpoint.
+  # Schemas endpoint returns configured value as it is.
+  config.schemas = [
+    # Define User schemas
+    {
+      # Basically, you don't have to change schemas/id/name/description
```

Comment:
```suggestion
      # Normally you don't have to change schemas/id/name/description
```
基本的にと言いたいと思うけど、普通は、の方が妥当かも